src/schedule/zephyr_ll.c

```diff
@@ -240,6 +241,12 @@ static void zephyr_ll_run(void *data)
 
 	zephyr_ll_unlock(sch, &flags);
 
+#if CONFIG_ZEPHYR_DP_SCHEDULER
+	/* trigger calculation of DP tasks as the last action of LL scheduler. Primary Core only*/
+	if (cpu_get_id() == PLATFORM_PRIMARY_CORE_ID)
+		scheduler_dp_ll_tick();
```

Comment:
hmm, now this makes me a bit uncomfortable... I think it would be better to make this just a normal LL task... Would that work? Just give it a priority 1 lower than all LL pipeline tasks